### ISSUE TITLE
train and validation loss are different for the same data

### ISSUE THREAD

**Issue by sabziequinor** [qubvel/segmentation_models#494]
Hi,

I am getting the strange results from segmentation models.
I am using the same data az train and validation, the loss ( accuracy metric ) are not following each other, in raining it is going down (up) and for validation it is going up (down), for loss (accuracy).
Is there any difference  between training and validation loss calculations?

Thanks in advance!
